test-files/ui/features/step_definitions/custom_steps.rb

```diff
@@ -760,3 +760,28 @@
 When(/^I click the review item column in the tag table$/) do
   page.all('td.f1').first.click
 end
+
+Then(/^I should see a node element "([^"]*)" with a selected highlight$/) do |id|
+  el = find('div.layer-data').first('g[class*=" ' + id + '"]')
+  el[:class].include? '.selected'
+end
+
+Then(/^I should see element "([^"]*)" with a yellow highlight$/) do |id|
+  el = find('div.layer-data').first('g[class*=" ' + id + '"]')
+  el[:class].include? '.edited.unsaved'
+end
+
+And(/^I should see an element with class "([^"]*)" on the map$/) do |id|
+  el = find('div.layer-data')
+  el[:class].include? id
```

Comment:
I think so? The way I understood [`include`](http://ruby-doc.org/core-1.9.3/Range.html#method-i-include-3F) was that it would search the element you select it on and everything below that.